Scripts/build-docs.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Generates the documentation for the project.
+# Usage: build-docs.sh <os-name>
+# Note: it must be run from Xcode.
+
+OS_NAME=$1
+PRODUCTS_DIR="${SRCROOT}/../WindowsAzureMessaging/${OS_NAME}"
+INSTALL_DIR="${PRODUCTS_DIR}/${PROJECT_NAME}.framework"
+DOCUMENTATION_DIR="${PRODUCTS_DIR}/Documentation/${PROJECT_NAME}"
+
+# Check if jazzy installed.
+if [ ! -x "$(command -v jazzy)" ]; then
+  echo "Couldn't find jazzy. Install jazzy before building frameworks"
+  exit 1
+fi
+
+jazzy --config "${SRCROOT}/../Documentation/${OS_NAME}/${PROJECT_NAME}/.jazzy.yaml"
```

Comment:
I do not see any `jazzy` configs in the repo, so I guess you do not need this script

Reply:
Yeah, planning on eventually adding docs but that's a longer term goal